ios/Classes/Module/FacebookModule.m

```diff
@@ -157,11 +157,21 @@ - (void)logPurchase:(NSArray<id> *_Nonnull)purchase
 {
   ENSURE_TYPE([purchase objectAtIndex:0], NSNumber);
   ENSURE_TYPE([purchase objectAtIndex:1], NSString);
+  ENSURE_TYPE([purchase objectAtIndex:2], NSDictionary);
 
   NSNumber *amount = [purchase objectAtIndex:0];
   NSString *currency = [TiUtils stringValue:[purchase objectAtIndex:1]];
+  NSDictionary *parameters = [purchase objectAtIndex:2];
 
-  [FBSDKAppEvents logPurchase:[amount doubleValue] currency:currency];
+  // [FBSDKAppEvents logPurchase:[amount doubleValue] currency:currency];
+  [FBSDKAppEvents logPurchase:[amount doubleValue] currency:currency parameters:parameters];
```

Comment:
If `parameters` is `nil`, either the other method should be used or you need to check if `nil` is okay to pass here.